kubernetes/couchdb/README.md

```diff
@@ -10,23 +10,33 @@ Pod. This can be done by running:
 kubectl apply -f couchdb.yml
 ```
 
-Once the Pod is up and running, the pod then needs to be
-configured. To do this, you need to run the configuration
-pod:
+This pod goes through the process of pulling the OpenWhisk
+repo and running through some of the ansible playbooks for
+configuring CouchDB.
+
+**NOTE** the pod will say running as soon as the start command
```

Comment:
I'll wait for CI to make sure it passes. If it doesn't I'll update everything in 1 PR.